pkg/objectupdate/rte/rte.go

```diff
@@ -136,7 +136,12 @@ func DaemonSetArgs(ds *appsv1.DaemonSet, conf nropv1.NodeGroupConfig) error {
 		return fmt.Errorf("cannot modify the arguments for container %s", cnt.Name)
 	}
 
-	flags.SetToggle("--refresh-node-resources")
+	rtePauseEnabled := isRTEPauseEnabled(&conf)
+	klog.V(2).InfoS("DaemonSet update: RTEPause status", "daemonset", ds.Name, "enabled", rtePauseEnabled)
+	if !rtePauseEnabled {
+		klog.V(4).InfoS("DaemonSet update: enable node resources refresh")
+		flags.SetToggle("--refresh-node-resources")
```

Comment:
this flag doesn't what you probably think: it asks the RTE to update the capacity (not just the current allocation): https://github.com/k8stopologyawareschedwg/resource-topology-exporter/blob/main/pkg/resourcemonitor/resourcemonitor.go#L404

it would be ok to add a new flag to RTE, but we will need a RTE PR